.github/workflows/stalebot.yml

```diff
@@ -16,25 +13,17 @@ jobs:
       issues: write
       pull-requests: write
     steps:
-      - name: Install Actions Core
-        run: npm --prefix .github/workflows/scripts install @actions/core
-
-      - name: Get start date
-        id: startdate
-        run: node .github/workflows/scripts/stalebot.js
```

Comment:
Since we're not using this script, can we delete it?

Reply:
Nice catch. Thanks for the reminder.